src/it/scala/org/apache/spark/sql/SFDataFrameAggregateSuite.scala

```diff
@@ -0,0 +1,89 @@
+package org.apache.spark.sql
```

Comment:
I know your general idea is to port SnowPark test code to Spark connector and you should add some change to make it works for spark connector. I think it will help the review if you can point out what's your change.